### ISSUE TITLE
Catch <script> and <?php in SVG files

### ISSUE THREAD

**Issue by gudmdharalds** [Automattic/VIP-Coding-Standards#178]
Catch  <script> and <?php in SVG files so that they can be flagged and they do not need to be reviewed manually.

**Comment by GaryJones** [comment]
One thing to note - by default, most projects will have a line in their PHPCS config file like:

```xml
	<arg name="extensions" value="php"/>
```

I think this would need to be changed to:

```xml
	<arg name="extensions" value="php,svg"/>
```

...for an SVG file to even have a chance of being checked. 

The TeamCity setup may need to be updated once the sniff is ready.

The [VIPSR script](https://gist.github.com/tomjn/2edadc0f9ba37e90a609945e25b42879#file-vipsr-sh-L350) does NOT use the `extensions` arg, so that should pick it up automatically once it's referenced in the `WordPress-VIP-Go` ruleset.

**Comment by gudmdharalds** [comment]
Closing this, as support for SVG checking will be moved in `vip-go-ci`, see #220.
